packaging/arch/PKGBUILD

```diff
@@ -195,4 +198,18 @@ package_snapd-git() {
   install -d -m 755 "$pkgdir/var/lib/snapd/seccomp/bpf"
   install -d -m 755 "$pkgdir/var/lib/snapd/snap/bin"
   install -d -m 755 "$pkgdir/var/lib/snapd/snaps"
+  install -d -m 755 "$pkgdir/var/lib/snapd/lib/gl"
```

Comment:
I think we have more than one directory now there (for opencl or for vulkan, not sure)